README.rst

```diff
@@ -1,9 +1,29 @@
-Papis
-=====
+.. raw:: html
 
-|ghbadge| |RTD| |CodeQL| |Pypi| |zenodo_badge|
+   <div align="center">
 
-Papis is a powerful and highly extensible CLI document and bibliography manager.
+   <img src="res/logo.svg" width=300>
```

Comment:
Maybe rename the folder to `resources`, so that it's clearer?

Otherwise, yeah, can't think of a better location for the logos.

Reply:
Ok, done!